membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -56,7 +68,7 @@ class AttributeController extends Controller with LazyLogging {
         request.touchpoint.attrService.get(id).map {
           case Some(attrs) =>
             onSuccess(attrs).withHeaders(
-              "X-Gu-Membership-Tier" -> attrs.Tier,
+              "X-Gu-Membership-Tier" -> attrs.Tier.get,
```

Comment:
if someone with an identity id calls this but not a member, will they get a 500 error?